int8 FrSky_controller_connection_status